test/Makefile

```diff
@@ -164,23 +164,30 @@ unit_tests: $(RESULTS_DIR)/unit_test_runner$(EXE)
 	@echo "Running unit tests"
 	$<
 
-run_runnable_tests: $(runnable_test_results)
+# start long-running tests as soon as possible for better parallelization
+run_runnable_tests: $(RESULTS_DIR)/runnable/sdtor.d.out $(RESULTS_DIR)/runnable/xtest46.d.out $(RESULTS_DIR)/runnable/test9259.d.out $(RESULTS_DIR)/runnable/test12.d.out $(RESULTS_DIR)/runnable/test17338.d.out $(RESULTS_DIR)/runnable/link2644.d.out $(runnable_test_results)
 
 start_runnable_tests: $(RESULTS_DIR)/.created $(test_tools)
 	@echo "Running runnable tests"
-	$(QUIET)$(MAKE) --no-print-directory run_runnable_tests
+	$(QUIET)$(MAKE) $(DMD_TESTSUITE_MAKE_ARGS) --no-print-directory run_runnable_tests
 
 run_compilable_tests: $(compilable_test_results)
 
 start_compilable_tests: $(RESULTS_DIR)/.created $(test_tools)
 	@echo "Running compilable tests"
-	$(QUIET)$(MAKE) --no-print-directory run_compilable_tests
+	$(QUIET)$(MAKE) $(DMD_TESTSUITE_MAKE_ARGS) --no-print-directory run_compilable_tests
 
-run_fail_compilation_tests: $(fail_compilation_test_results)
+run_fail_compilation_tests: $(RESULTS_DIR)/fail_compilation/fail12485.sh.out $(fail_compilation_test_results)
```

Comment:
Why is this change required? (i.e. why `.out` and not `.sh`)

Reply:
This just makes sure `fail12485.sh` is started first in the `fail_compilation` tests block, as it takes about as much time as all other fail tests combined (& parallelized).

Reply:
Can we add this as a comment so that people in the future aren't confused? Thanks!

Reply:
It's 13 lines up, I assumed it'd be clear when viewing the whole file.

Reply:
I totally missed Iain's #9270; this here is derived from 2.084-based LDC - updated accordingly.